empiric-package/empiric/core/types.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+from typing_extensions import Literal
+
+ADDRESS = int
+HEX_STR = str
+
+# Network Types
+STAGING = "staging"
+TESTNET = "testnet"
+INTEGRATION = "https://external.integration.starknet.io"
+MAINNET = "mainnet"
+
+Network = Literal[
+    "staging", "testnet", "https://external.integration.starknet.io", "mainnet"
```

Comment:
The integration url is necessary with the current setup. Not very clean though tbh. Gonna make a PR to explore a different option.